packages/react-label/src/components/Label/useLabelStyles.ts

```diff
@@ -7,8 +7,8 @@ import { LabelState } from './Label.types';
 const useStyles = makeStyles({
   root: theme => ({
     fontFamily: theme.global.type.fontFamilies.base,
-    fontSize: theme.global.type.fontSizes.base[300],
```

Comment:
from the PR description, was the removal of `fontSize` intentional ?

Reply:
Yes, since all sizes have their own `fontSize` and size has a default of `medium`.